src/Inciter/DG.C

```diff
@@ -39,6 +41,7 @@ DG::DG( const CProxy_Discretization& disc, const tk::CProxy_Solver& solver ) :
 {
   // Signal the runtime system that the workers have been created
   solver.ckLocalBranch()->created();
+  (void)fd;
```

Comment:
For future reference: this could also be done using the macro `IGNORE`, defined in `Base/Macro.h`. It does the same as you did, but using C++'s `static_cast`.

Reply:
Okay, made this change anyway.